package.json

```diff
@@ -13,14 +13,15 @@
   "scripts": {
     "codegen": "scripts/codegen.sh",
     "build": "scripts/build.sh",
+    "acceptance": "scripts/acceptance.sh",
     "publish": "scripts/publish.sh",
     "fix": "run-s fix:*",
     "fix:prettier": "prettier \"src/**/*.ts\" --write",
     "fix:lint": "eslint src --ext .ts --fix",
-    "jest": "jest",
+    "jest": "TZ=America/New_York jest",
```

Comment:
why is this needed?

Reply:
The test mock responses were built for the EST timezone, so running with a different timezone fails.

See this test failure: https://github.com/getoctane/octane-node/runs/3616483350?check_suite_focus=true

cc @salemhilal